service/src/main/java/uk/gov/hmcts/reform/fpl/model/notify/SharedNotifyTemplate.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.hmcts.reform.fpl.model.notify;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.List;
+
+@Getter
+@Setter
+public class SharedNotifyTemplate implements NotifyData {
+    private List<String> ordersAndDirections;
+    private String dataPresent;
+    private String fullStop;
+    private String timeFramePresent;
+    private String timeFrameValue;
+    private String urgentHearing;
+    private String nonUrgentHearing;
+    private String firstRespondentName;
+    private String reference;
+    private String caseUrl;
+    private String localAuthority;
```

Comment:
If we make all of these protected/package private then we don't need a getter and can just call the values directly in the other classes